cmake/scripts/webos/Install.cmake

```diff
@@ -67,6 +67,10 @@ add_custom_command(
   VERBATIM
 )
 
+file(TOUCH dummy.c)
+add_library(AcbAPI SHARED dummy.c)
+set_target_properties(AcbAPI PROPERTIES VERSION 1.0.0 SOVERSION 1)
```

Comment:
I'd move this to the find module, for the lack of a better place

Reply:
Moved it over